ovirt/resource_datecenter.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2017 Battelle Memorial Institute
+// All rights reserved.
+//
+// This software may be modified and distributed under the terms
+// of the BSD-2 license.  See the LICENSE file for details.
+
+package ovirt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	ovirtsdk4 "gopkg.in/imjoey/go-ovirt.v4"
+)
+
+func resourceDataCenter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDataCenterCreate,
+		Read:   resourceDataCenterRead,
+		Delete: resourceDataCenterDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"local": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceDataCenterCreate(d *schema.ResourceData, meta interface{}) error {
+
+	conn := meta.(*ovirtsdk4.Connection)
+
+	DataCenterBuilder := ovirtsdk4.NewDataCenterBuilder().
+		Name(d.Get("name").(string)).
+		Description(d.Get("description").(string)).
+		Local(d.Get("local").(bool)).
```

Comment:
For optional attributes of boolean type, need to call `d.GetOkExists(key)` method to see if it has been set or not.